TFT/src/User/config.ini

```diff
@@ -27,8 +28,8 @@
 # Distribution date: 2021-05-15
 # Firmware source: https://github.com/MarlinFirmware/Marlin/releases
 #
-# In order the TFT firmware is able to provide all of its functionalities/features,
-# ensure that the following options are enabled in Marlin firmware.
+# In order for the TFT's firmware to be able to provide all of its functionalities/features,
+# the following options must be enabled in Marlin firmware.
 #
```

Comment:
It will be much better to keep it straightforward like:
`To use all the features & functionalities supported by the TFT, the following options must be enabled in Marlin firmware.`